06-files-input-and-output/remove_files.py

```diff
@@ -1,49 +0,0 @@
-# Copyright (c) 2016 Ami . All rights reserved
```

Comment:
These files *are* in master. If I merge this branch they will be deleted too. Talk to me f2f so we can resolve this.